ngraph/core/include/ngraph/op/roi_align.hpp

```diff
@@ -76,6 +76,9 @@ namespace ngraph
                 int get_sampling_ratio() const { return m_sampling_ratio; }
                 float get_spatial_scale() const { return m_spatial_scale; }
                 PoolingMode get_mode() const { return m_mode; }
+                bool evaluate(const HostTensorVector& outputs,
```

Comment:
Just the general question. At the current moment we have a recommendation to don't add evaluate method to operation if we use it only in tests. Do we have real topology where we need to call evaluate for this topology?

Reply:
AFAIR this op's evaluator was requested by the VPU team for validation purposes.

Reply:
I think this op is required to support Mask-RCNN, there is a jira ticket for that: 23134.